vmdk_plugin/Makefile

```diff
@@ -101,7 +101,9 @@ SRC = main.go log_formatter.go utils/refcount/refcnt.go \
 
 TEST_SRC = ../tests/utils/inputparams/testparams.go \
 	../tests/utils/dockercli/volumelifecycle.go \
-	../tests/utils/verification/volumeproperties.go
+	../tests/utils/verification/volumeproperties.go \
```

Comment:
Did it in alignment with other utils (line 102,103).  required to build test files. If not required, should we remove all of them?